Tests/Packet++Test/PacketExamples/BgpUpdate.dat

```diff
@@ -0,0 +1 @@
+00c04f23c59500000c350e1c080045c0006800050000ff06394ac0a80021c0a8000f00b3084c7a40e076d6339e6a50183ef81d7e0000ffffffffffffffffffffffffffffffff0040020000002740010101400200400304c0a800218004040000000040050400000064c00808fe0901f4fe090258080a
```

Comment:
It'd be good to add the pcap file you are using in addition to the .dat files. You can add just 1 pcap file that contains all of the packets you're using for testing